docs/source/intro/install_conda.rst

```diff
@@ -84,6 +84,12 @@ This is the recommended method for users on shared systems and systems where the
        Last name: name
        Institution: where-i-work
 
+   .. admonition:: Having problems auto-detecting the PostgreSQL setup?
+       :class: attention title-icon-troubleshoot
+
+       Sometimes the local configuration of postgres may be difficult to auto-detect for AiiDA.
```

Comment:
You capitalized postgres in the title. If we do it there, we should it everywhere.